src/helpers/isFilled.ts

```diff
@@ -248,6 +249,17 @@ export const geoPoint = (
 	return isNonNullish(field) && "longitude" in field
 }
 
+/**
+ * Determines if a table field is filled.
+ *
+ * @param field - Table field to check.
+ *
+ * @returns `true` if `field` is filled, `false` otherwise.
+ */
+export const table = isNonNullish as (
```

Comment:
we might want to do something more sophisticated here, i.e. a table with no head and an empty body is probably "not filled"

Reply:
It's true that by default, we will define content for ALL tables, and we will not allow full delete, meaning we will mostly have content for every new pages containing a table. Just with empty cells. Personally, I'm ok to take a table with only empty cell as just a **filled** table field, as even empty cell means table content is defined in my mind. WDYT?

Reply:
the question is whether as a developer you want to show something on the screen or not if you have a table content defined but with empty rows/cells

for the marketer, having tabl content defined with empty cells means there is content, imo, because there's structure that maybe they want to fill later

but for the consumer side, for the developer, i'm not so sure

Reply:
Discussed with Lucie, it's ok not to check every rows / cells to verify the content of the table. But it could be good to add a comment here about the fact table field with rows is considered filled. Even if each cells are an empty text.

Reply:
Fine as is to me yes!